soap/src/java/com/zimbra/soap/admin/message/GetAccountRequest.java

```diff
@@ -53,6 +53,17 @@
     @XmlAttribute(name=AdminConstants.A_APPLY_COS, required=false)
     private ZmBoolean applyCos = ZmBoolean.ONE /* true */;
 
+    /**
+     * @zm-api-field-tag effectiveQuota
+     * @zm-api-field-description Flag whether want to return effective value (minimum of zimbraMailQuota & zimbraMailDomainQuota)
```

Comment:
Idea, referring the description of applyCos
```suggestion
     * @zm-api-field-description Flag whether or not to get effective value (minimum of zimbraMailQuota and zimbraMailDomainQuota)
```

Reply:
Done